src/js/common/components/Modal.jsx

```diff
@@ -78,7 +78,7 @@ class Modal extends React.Component {
     }
 
     return (
-      <div className={modalCss} id={id} role="alertdialog" aria-labelledby={`${id}-title`}>
+      <div className={modalCss} id={id} role="alertdialog" aria-labelledby={id && title ? `${id}-title` : undefined}>
```

Comment:
Should the `id` prop be required so that this is always defined? I found at least one place where it's not passed in (https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/rx/components/DiscardChangesModal.jsx#L36)  but it doesn't seem to be tested in e2e tests.